mongodbatlas/resource_mongodbatlas_alert_configuration.go

```diff
@@ -670,11 +680,17 @@ func flattenAlertConfigurationThresholdConfig(m *matlas.Threshold) []interface{}
 	return []interface{}{}
 }
 
-func expandAlertConfigurationNotification(d *schema.ResourceData) []matlas.Notification {
+func expandAlertConfigurationNotification(d *schema.ResourceData) ([]matlas.Notification, error) {
 	notifications := make([]matlas.Notification, len(d.Get("notification").([]interface{})))
 
 	for i, value := range d.Get("notification").([]interface{}) {
 		v := value.(map[string]interface{})
+		if v1, ok := v["interval_min"]; ok && v1.(int) > 0 {
+			typeName := v["type_name"].(string)
+			if strings.EqualFold(typeName, "pager_duty") || strings.EqualFold(typeName, "ops_genie") || strings.EqualFold(typeName, "victor_ops") {
+				return nil, fmt.Errorf(`'interval_min' don't need to be set if type_name is 'PAGER_DUTY', 'OPS_GENIE' or 'VICTOR_OPS'`)
```

Comment:
`'interval_min' does't need to be set if type_name is 'PAGER_DUTY', 'OPS_GENIE' or 'VICTOR_OPS'

Reply:
Anyways don't like this wording... So I'll think of another one

Reply:
Ok will update, thank you